libs/model/src/services/commonProtocol/contestHelper.ts

```diff
@@ -55,17 +55,28 @@ export const addContent = async (
   contest: string,
   creator: string,
   url: string,
+  web3?: Web3,
+  nonce?: number,
```

Comment:
guess we are exposing the nonce implementation details after all 😆

Reply:
FWIW, the batch funcs don't expose it– these old funcs can be treated as private/internal now

Reply:
just trying to follow the open-closed principle, but guess we can deal with the potential debt of opening these details when we start accumulating it in the future

Reply:
agreed– but in the context of the batching, these args are actually needed to get the nonce + shared web3 instance from the batch caller

we just need to ensure that the single-contest funcs remain unexposed and cannot be used outside the helper lib.

Reply:
ok, so are we removing the export clause?

Reply:
These functions will likely be pushed in @rbennettcw work so feel free to open/close functions as desired based on what required. As mentioned if we're always calling batch version the old versions dont need to be exported but will allow existing work to refactor to ensure that doesnt break any other current work